sklearn/metrics/tests/test_common.py

```diff
@@ -446,10 +475,12 @@ def test_sample_order_invariance_multilabel_and_multioutput():
 
     for name in MULTILABELS_METRICS:
         metric = ALL_METRICS[name]
-        assert_almost_equal(metric(y_true, y_pred),
-                            metric(y_true_shuffle, y_pred_shuffle),
-                            err_msg="%s is not sample order invariant"
-                                    % name)
+        if name != 'unnormalized_jaccard_similarity_score' and \
```

Comment:
I've removed `none-samples` jaccard_similarity_score from the test of *sample order invariance*.

Reply:
But the unnormalized score is order invariant isn't it?

I'm feeling like backtracking on the none-samples idea. It's useful, but the user can get it by just transposing the input. It's not that helpful towards an unnormalized score.

Reply:
Actually `average='sample'` with `normalize=False` was changed to have the same behavior as `average='none-samples'` (including to return an array of scores). So it is *now*, **not** sample-order invariant.

So, you mean we just remove average='none-samples' as a possible value of `average` but retain `normalize` (i.e not deprecate it), and for `normalize=False` return a score (instead of an array)?

Or do we want to remove `normalize` from `jaccard_similarity_score` API?

Though I think we should prefer to retain `normalize` (i.e. not deprecate it).